### PR TITLE
added armis-api

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    <img src={useBaseUrl('img/send-data/armis-settings.png')} alt="armis-settings.png" width="900" />
+1. Create a new API secret key if you haven't created yet. Click **Create** button from API Management page.
+
+    <img src={useBaseUrl('img/send-data/create-api.png')} alt="create-api.png" width="=700" />
+
+1. Click **Show** to view the secret key.
+
+   <img src={useBaseUrl('img/send-data/show-secretkey.png')} alt="show-secretkey.png" width="700" />
+
+1. A popup window will be displayed, copy and paste the secret key to a folder location. Remember, you will need to enter this key while creating the **Armis Cloud-to-Cloud Source**.  <img src={useBaseUrl('img/send-data/show-key.png')} alt="show-key.png" width="400" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:61]:
```suggestion
1. Navigate to **Settings** > **API Management** on your Armis application.<br/> <img src={useBaseUrl('img/send-data/armis-settings.png')} alt="armis-settings.png" width="900" />
1. Create a new API secret key if you haven't created one yet by clicking the **Create** button from the API Management page.<br/> <img src={useBaseUrl('img/send-data/create-api.png')} alt="create-api.png" width="=700" />
1. Click **Show** to view the secret key.<br/> <img src={useBaseUrl('img/send-data/show-secretkey.png')} alt="show-secretkey.png" width="700" />
1. A popup window will be displayed, copy and paste the secret key to a folder location. Remember, you will need to enter this key while creating the **Armis Cloud-to-Cloud Source**.  <img src={useBaseUrl('img/send-data/show-key.png')} alt="show-key.png" width="400" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    <img src={useBaseUrl('img/send-data/armis-icon.png')} alt="armis-icon.png" width="150" />
+
+1. Enter a **Name** for the Source. The description is optional.
+    <img src={useBaseUrl('img/send-data/armis-config-main.png')} alt="armis-config-main" width="450" />
+
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate, each field needs a name (key) and value.
+
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:79]:
```suggestion
1. Select **Armis**.<br/> <img src={useBaseUrl('img/send-data/armis-icon.png')} alt="armis-icon.png" width="150" />
1. Enter a **Name** for the Source. The description is optional.<br/> <img src={useBaseUrl('img/send-data/armis-config-main.png')} alt="armis-config-main" width="450" />
1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
1. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored (i.e., dropped).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+1. In **Secret Key**, enter your API secret key you have generated in [Setup and Configuration](#setup-and-configuration) section.
+1. In **Armis API selection**. Choose the data sources from which you want to ingest data. The integration provides the option to you to select either one or both of the data sources.
+
+* If **Alert API** is selected, the integration will fetch alerts data.
+  * Permission `Alert>Read` must be provided to fetch alerts data.
+  * Data for alert will be fetched every 5 minutes.
+
+* If **Device API** is selected, the integration will fetch device data.
+  * Permission `Device>Read` must be provided to fetch device data.
+  * Data for device will be fetched every 24 hours.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:91]:
```suggestion
1. In **Instance URL**, enter the Armis hostname. For example, https://armis-instance.armis.com.
 :::note
 Armis Instance URL is the Armis hostname.
 :::
1. In **Secret Key**, enter your API secret key you have generated in [Setup and Configuration](#setup-and-configuration) section.
1. In **Armis API selection**. Choose the data sources from which you want to ingest data. The integration provides the option to you to select either one or both of the data sources.
   * If **Alert API** is selected, the integration will fetch alerts data.
     * Permission `Alert>Read` must be provided to fetch alerts data.
     * Data for alert will be fetched every 5 minutes.
   * If **Device API** is selected, the integration will fetch device data.
     * Permission `Device>Read` must be provided to fetch device data.
     * Data for device will be fetched every 24 hours.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+:::note
+This step is mandatory, ensure you select one data source.
+:::
+1. (Optional) In **Processing Rules for Logs**, configure any desired filters, such as allow list, deny list, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+
+1. When you are finished configuring the Source, click **Save**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:94]:
```suggestion
:::note
This step is mandatory, ensure you select one data source.
:::
1. (Optional) In **Processing Rules for Logs**, configure any desired filters, such as allow list, deny list, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
1. When you are finished configuring the Source, click **Save**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+1. Select **Armis**.<br/> <img src={useBaseUrl('img/send-data/armis-icon.png')} alt="armis-icon.png" width="150" />
+1. Enter a **Name** for the Source. The description is optional.<br/> <img src={useBaseUrl('img/send-data/armis-config-main.png')} alt="armis-config-main" width="450" />
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored (i.e., dropped).
+
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:68]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+   * If **Alert API** is selected, the integration will fetch alerts data.
+     * Permission `Alert>Read` must be provided to fetch alerts data.
+     * Data for alert will be fetched every 5 minutes.
+   * If **Device API** is selected, the integration will fetch device data.
+     * Permission `Device>Read` must be provided to fetch device data.
+     * Data for device will be fetched every 24 hours.
+:::note
+This step is mandatory, ensure you select one data source.
+:::
+1. (Optional) In **Processing Rules for Logs**, configure any desired filters, such as allow list, deny list, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:83]: allow list, deny list are one word (_allowlist_, _denylist_)

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md:83]: allow list, deny list are one word (_allowlist_, _denylist_)

```suggestion
1. (Optional) In **Processing Rules for Logs**, configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md]: @kimsauce Thank you for the suggestions.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**